packages/flutter/test/widgets/platform_menu_bar_test.dart

```diff
@@ -233,27 +185,27 @@ void main() {
     testWidgets('character constructor', (WidgetTester tester) async {
       final ShortcutSerialization serialization = ShortcutSerialization.character('?');
       expect(serialization.toChannelRepresentation(), equals(<String, Object?>{
-        'shortcutCharacter': '?',
-        'shortcutModifiers': 0,
-      }));
+            'shortcutCharacter': '?',
+            'shortcutModifiers': 0,
+          }));
```

Comment:
It's hard to tell on GitHub, but are these indentations intentional?  I thought they looked correct beforehand.

Reply:
No, I think VSCode's auto format went rogue. I'll see if I can configure it to match the previous formatting.